README.Rmd

```diff
@@ -31,11 +31,10 @@ running PACTA for Banks, in the form of a [cookbook](https://rmi-pacta.github.io
 
 ## Installation
 
-You can install the development version of `{pacta.loanbook}` from [GitHub](https://github.com/) with:
+You can install the release version of the package by running the following command in R:
 
 ``` r
-# install.packages("pak")
-pak::pak("RMI-PACTA/pacta.loanbook")
+install.packages("pacta.loanbook")
```

Comment:
{usethis} explicitly suggests to update install instructions ("aspirationally") in the README